v3/src/components/graph/models/data-configuration-model.ts

```diff
@@ -144,6 +144,14 @@ export const DataConfigurationModel = types
     placeShouldShowClickHereCue(place: GraphPlace, tileHasFocus: boolean) {
       return this.placeAlwaysShowsClickHereCue(place) ||
         (this.placeCanShowClickHereCue(place) && tileHasFocus)
+    },
+    isCaseInSubPlot(subPlotKey: Record<string, string>, caseData: Record<string, any>) {
+      const numOfKeys = Object.keys(subPlotKey).length
+      let matchedValCount = 0
+      Object.keys(subPlotKey).forEach(key => {
+        if (subPlotKey[key] === "" || subPlotKey[key] === caseData[key]) matchedValCount++
```

Comment:
```suggestion
        if (subPlotKey[key] === caseData[key]) matchedValCount++
```
I think if we make the change I suggested above about not including empty strings in subplot keys _then_ we wouldn't need this test here.